pkg/arvo/app/landscape/js/channel.js

```diff
@@ -71,9 +75,16 @@ class Channel {
     });
   }
 
+  ack() {
+    if(this.lastAcknowledgedEventId === this.lastEventId) {
+      return;
+    }
+    this.sendJSONToChannel();
+  }
+
   //  sends a poke to an app on an urbit ship
   //
-  poke(ship, app, mark, json, successFunc, failureFunc) {
+  poke(ship, app, mark, json, successFunc, failureFunc, queue = false) {
```

Comment:
I don't think this 'queue' variable should be a part of Channel.js's API. I think we should either handle the enqueue and dequeue logic *entirely* in Channel.js (and opaquely), or entirely in Landscape. I lean towards Channel.js handling the enqueue and dequeue logic completely. Mixing the responsibilities of Landscape and Channel should be avoided.